kore/src/Kore/Step/Simplification/Inj.hs

```diff
@@ -0,0 +1,51 @@
+{- |
+Copyright   : (c) Runtime Verification, 2019
+License     : NCSA
+
+-}
+module Kore.Step.Simplification.Inj
+    ( simplify
+    ) where
+
+import Data.Functor.Compose
+
+import Kore.Internal.Condition as Condition
+import Kore.Internal.Inj
+    ( Inj (..)
+    )
+import Kore.Internal.MultiOr
+    ( MultiOr
+    )
+import Kore.Internal.OrPattern
+    ( OrPattern
+    )
+import Kore.Internal.TermLike
+import Kore.Step.Simplification.InjSimplifier
+    ( InjSimplifier (..)
+    )
+import Kore.Step.Simplification.Simplify as Simplifier
+
+{- |'simplify' simplifies an 'Inj' of 'OrPattern'.
+
+-}
+simplify
+    :: (SimplifierVariable variable, MonadSimplify simplifier)
+    => Condition variable
```

Comment:
I guess it's fine if you want to leave this here, but we have simplifiers that do not take a `Condition` (e.g. `Top.simplify`), so you won't have a uniform interface for all simplifiers anyway, and removing it would mean that this function takes as input only what's actually needed.